terraform/workspace-variables/production.tfvars

```
@@ -12,7 +12,8 @@ paas_web_app_deployment_strategy = "blue-green-v2"
 paas_web_app_instances = 4
 paas_web_app_memory = 8192
 paas_worker_app_instances = 1
-paas_worker_app_start_command = "/app/bin/delayed_job --pool=mailers --pool=*:2 start && bundle exec rake jobs:work"
+paas_worker_app_start_command = "/app/bin/delayed_job --pool=mailers --pool=*:2 start && (bundle exec rake jobs:work &) && bundle exec sidekiq -C config/sidekiq.yml"
+paas_redis_service_plan = "tiny-ha-5_x"
```

```suggestion
paas_redis_service_plan = "tiny-ha-6_x"
```

or even

```suggestion
paas_redis_service_plan = "small-ha-6_x"
```

What do you think? If you accept this, please use 6_x for all environments.